### ISSUE TITLE
Any way to record the webinar? 

### ISSUE THREAD

**Issue by jashasweejena** [githubschool/funny-lemur#53]
# @crichID

**Comment by crichID** [comment]
@jashasweejena we will send you a recording after the class!

**Comment by jashasweejena** [comment]
Thank you :+1:

**Comment by jashasweejena** [comment]
I just wanted to know if you are gonna send me the recordings after tomorrow's session? I missed today's session, so i was curious about watching those :sweat_smile: 

Thanks in advance  :smile:

**Comment by crichID** [comment]
@jashasweejena yes, we will send the videos after tomorrow's session so you will be able to catch up on what you missed today!

**Comment by jashasweejena** [comment]
Thank you :smile:
